crates/tuono/src/typescript/file_types.rs

```diff
@@ -0,0 +1,89 @@
+use super::parser;
+use super::utils::has_derive_type;
+use std::error::Error;
+use std::path::PathBuf;
+use tracing::trace;
+
+/// Represents all the valid typescript types found in a file.
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
We may not need `Debug` in production
```suggestion
#[derive(Clone, PartialEq, Eq)]
```